libs/blocks/quiz-results/quiz-results.js

```diff
@@ -54,7 +54,13 @@ export default async function init(el, debug = null, localStoreKey = null) {
 
   /* eslint-disable no-param-reassign */
   // handle these two query param values in this way to facilitate unit tests
-  localStoreKey ??= params.get('quizKey');
+  localStoreKey ??= params.get('quizkey');
+
+  const { locale } = getConfig();
+  if (locale?.ietf) {
```

Comment:
Rather than multi-line statements, folks seem to prefer single-line variants to be as obscure as possible. Would something like this variation work? 
localStoreKey = locale?.ietf ? `${localStoreKey}-${locale.ietf}` : localStoreKey;